includes/class-fontawesome.php

```diff
@@ -1363,8 +1370,25 @@ function( $html, $handle ) use ( $resource_collection ) {
 				);
 
 				if ( $load_spec['v4shim'] ) {
-					// phpcs:ignore WordPress.WP.EnqueuedResourceParameters
-					wp_enqueue_style( self::RESOURCE_HANDLE_V4SHIM, $resource_collection[1]->source(), null, null );
+					add_action(
+						'wp_enqueue_scripts',
```

Comment:
I experimented with the enqueue priority, earlier or later, and it didn't seem to make a difference. I think I may have overlooked something. I'm concerned that we may still be getting false positives in this draft, but I'll probably need help to know just what scenario I can reproduce that would properly exercise the load order concern.

Reply:
I don't know enough about this to comment.